### PR TITLE
Feature/producer consumer

### PR DESCRIPTION
**Pull request by bhaskarkh**
added Sample Producer  feature . we can invoke a message t Kafka on api call

### CODE REVIEW

File: src/main/java/com/bhaskar/kafka/KafkaApplication.java [hunk 1]
```diff
@@ -10,4 +13,8 @@ public static void main(String[] args) {
 		SpringApplication.run(KafkaApplication.class, args);
 	}
 
+  @Bean
```
[the remark this answers is not in the document]
Comment (bhaskarkh) [reply src/main/java/com/bhaskar/kafka/KafkaApplication.java]: removed

**Review by bhaskarkh — COMMENTED** [review]
good to merge

**Review by bhaskarkh — COMMENTED** [review]
issue  ressolved

**Review by CodingErrorSolution — APPROVED** [review]
good to merge.  need to do cleanup in next PR

**Pull request merged**